dev-java/ant-eclipse-ecj/ant-eclipse-ecj-4.15-r1.ebuild

```diff
@@ -21,10 +21,10 @@ IUSE=""
 CDEPEND="~dev-java/eclipse-ecj-${PV}:${SLOT}
 	dev-java/ant-core:0"
 RDEPEND="${CDEPEND}
-	>=virtual/jre-11"
+	>=virtual/jre-11:*"
 DEPEND="${CDEPEND}
 	app-arch/unzip
```

Comment:
i suppose this should go to `BDEPEND`.

Reply:
Why do you think this one should be a `BDEPEND`?

I'm assuming the original ebuild author got it rigth, that this is a runtime dependency. Wouldn't the `BDPEPEND` get depcleaned and then cause runtime issues?

Reply:
it's there for unpacking the jar file. i can't see no ocurrance of `unzip` in the sources.

Reply:
got it, thanks for your explanation. I fixed this.